NEWS

```diff
@@ -2,6 +2,56 @@ News
 ====
 
 
+Version 3.0.0 rc1
+-------------------
+
+Since releasing 3.0.0 beta1 in June no critical issues have surfaced. This
+release includes some feature and compability improvements but no breaking
+changes.
+
+Upgrade notes
+~~~~~~~~~~~~~
+
+This release comes with the same restrictions as the previous one: to upgrade,
+you either need 2.16.2 or 3.0.0 beta1 installed. Upgrading directly from older
+versions or from the Ganeti-2.17 beta version is not supported. Please refer
+to the 3.0.0 beta1 upgrade notes for more information.
+
+Important changes
+~~~~~~~~~~~~~~~~~
+
+GHC 8.0 through 8.8 compatibility
++++++++++++++++++++++++++++++++++
+
+This release has been built/tested against GHC 8.0 through 8.8 which means it
+should work on most current and near-future distribution versions. Support for
+GHC versions < 8 has already been dropped with the previous Ganeti release. Along
+with this change we have also added compatibility to Cabal version 3.0.
+
+Other notable changes
+~~~~~~~~~~~~~~~~~~~~~
+
+Bugfixes:
+ - Fix distribution of hmac.key to new nodes - this has been pulled from the 2.17 tree #(1494)
+
+Compatibility Improvements:
+ - Open vSwitch: Do not fail to add node when the ovs_name or ovs_link already exists (#1495)
+ - Improved support for DRBD => 8.4 (#1496)
```

Comment:
`=>` → `>=` :-)

Reply:
Or ≥ ? (somewhat joking, just that I'm an Unicode fan)